### PR TITLE
journald -> sumologic pipe: I'm a plumber!

### PR DESCRIPTION
**Pull request by chr0n1x**

### CODE REVIEW

File: v2/fleet/sumologic-journald.service [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ExecStartPre=-/usr/bin/docker kill sumologic-journald
+ExecStartPre=-/usr/bin/docker rm -f sumologic-journald
+ExecStart=/usr/bin/bash -c \
+"if [[ -f /etc/profile.d/etcdctl.sh ]]; then source /etc/profile.d/etcdctl.sh;fi && \
+sudo /usr/bin/docker run --name sumologic-journald \
+-p 514:514 \
+-p 514:514/udp \
+-e PATH_EXPRESSION=/syslog/syslog \
+-e SUMO_NAME=${NODE_PRODUCT}-${NODE_TIER}-cluster \
+-e SUMO_CATEGORY=be/${NODE_PRODUCT}/${NODE_TIER}/journald-logs \
```
Comment (chrishenry) [inline v2/fleet/sumologic-journald.service]: Will we be able to filter journald logs for control, proxy and worker ASGs with this sourceCategory?

Comment (chr0n1x) [inline v2/fleet/sumologic-journald.service]: nope, will add that.

**Comment by mannytoledo** [comment]
:+1:

File: v2/fleet/sumologic-journald.service [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ExecStartPre=-/usr/bin/docker kill sumologic-journald
+ExecStartPre=-/usr/bin/docker rm -f sumologic-journald
+ExecStart=/usr/bin/bash -c \
+"if [[ -f /etc/profile.d/etcdctl.sh ]]; then source /etc/profile.d/etcdctl.sh;fi && \
+sudo /usr/bin/docker run --name sumologic-journald \
+-p 514:514 \
+-p 514:514/udp \
+-e PATH_EXPRESSION=/syslog/syslog \
+-e SUMO_NAME=${NODE_PRODUCT}-${NODE_TIER}-cluster \
+-e SUMO_CATEGORY=be/${NODE_PRODUCT}/${NODE_TIER}/${NODE_ROLE}/journald-logs \
```
Comment (chrishenry) [inline v2/fleet/sumologic-journald.service:22]: :+1: :roller_coaster:

**Pull request merged**